ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.15__okd.yaml

```diff
@@ -0,0 +1,68 @@
+base_images:
+  openshift_release_golang-1.20:
+    name: release
+    namespace: openshift
+    tag: golang-1.20
+binary_build_commands: make build
+build_root:
+  from_repository: true
+  use_build_cache: true
+images:
+- dockerfile_path: build/Dockerfile.ci
+  inputs:
+    openshift_release_golang-1.20:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.20
+  to: windows-machine-config-operator-test
+releases:
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.15"
+      namespace: origin
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 400Mi
+  src:
+    limits:
+      memory: 10Gi
+    requests:
+      cpu: 100m
+      memory: 10Gi
+  windows-machine-config-operator-test:
+    limits:
+      memory: 10Gi
+    requests:
+      cpu: 100m
+      memory: 10Gi
+tests:
+- as: lint
+  commands: make lint
+  container:
+    from: src
+- as: build
+  commands: make build
+  container:
+    from: src
+- as: unit
+  commands: make unit
+  container:
+    from: src
```

Comment:
I suspect we don't need these given there are no PRs opened and no presubmit tests. Not sure why they are present in other community operator configs.

Reply:
We had them when we did community backports, but not needed anymore.